pom.xml

```diff
@@ -229,6 +229,10 @@
 			<artifactId>scenery</artifactId>
 			<version>0.1.1-SNAPSHOT</version>
 		</dependency>
+		<dependency>
+			<groupId>net.imagej</groupId>
+			<artifactId>imagej-legacy</artifactId>
```

Comment:
Mostly concerns of my group running out of funding and students not being able to finish their degrees if I don't get enough preliminary data/demos to use in grants that have yet to be written. Otherwise I'd be happy to.